src/styles/styles.css

```diff
@@ -0,0 +1,127 @@
+body {
```

Comment:
Why is this file has .css extension? 
It has to be .scss for you to use nesting. 

You have it in the task description + I have asked you twice (with an example) to use nesting here. 
Read: https://uxplanet.org/nesting-techniques-in-scss-1699833baca1